presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergQueryRunner.java

```diff
@@ -282,39 +282,7 @@ public static void main(String[] args)
             throws Exception
     {
         setupLogging();
-        Optional<Path> dataDirectory = Optional.empty();
-        if (args.length > 0) {
-            if (args.length != 1) {
-                log.error("usage: IcebergQueryRunner [dataDirectory]\n");
-                log.error("       [dataDirectory] is a local directory under which you want the iceberg_data directory to be created.]\n");
-                System.exit(1);
-            }
-
-            File dataDirectoryFile = new File(args[0]);
-            if (dataDirectoryFile.exists()) {
-                if (!dataDirectoryFile.isDirectory()) {
-                    log.error("Error: " + dataDirectoryFile.getAbsolutePath() + " is not a directory.");
-                    System.exit(1);
-                }
-                else if (!dataDirectoryFile.canRead() || !dataDirectoryFile.canWrite()) {
-                    log.error("Error: " + dataDirectoryFile.getAbsolutePath() + " is not readable/writable.");
-                    System.exit(1);
-                }
-            }
-            else {
-                // For user supplied path like [path_exists_but_is_not_readable_or_writable]/[paths_do_not_exist], the hadoop file system won't
-                // be able to create directory for it. e.g. "/aaa/bbb" is not creatable because path "/" is not writable.
-                while (!dataDirectoryFile.exists()) {
-                    dataDirectoryFile = dataDirectoryFile.getParentFile();
-                }
-                if (!dataDirectoryFile.canRead() || !dataDirectoryFile.canWrite()) {
-                    log.error("Error: The ancestor directory " + dataDirectoryFile.getAbsolutePath() + " is not readable/writable.");
-                    System.exit(1);
-                }
-            }
-
-            dataDirectory = Optional.of(dataDirectoryFile.toPath());
-        }
```

Comment:
I don't really see why we need to remove the ability to specify the data directory through program arguments. I have good number of IDE run configurations which utilize the program arguments already. Can't we just add the ability to read from the env variables if program arguments aren't specified?

In my experience designing an application that takes some number of configuration values, they should be resolved in the following priority (higher options take precedence over lower ones)

1. CLI arguments
2. environment variables
3. user config: e.g. read from `$HOME/.config` or similar
4. system config e.g. `/etc/....`

Reply:
Makes sense, will do the changes.